.github/workflows/hubble-cli.yaml

```diff
@@ -0,0 +1,147 @@
+    steps:
+      - name: Check code changes
+        uses: dorny/paths-filter@de90cc6fb38fc0963ad72b210f1f284cd68cea36 # v3.0.2
+        id: tested-tree
+        with:
+          filters: |
+            src:
+              - '!(test|Documentation)/**'
+
+  build-hubble-cli-release-binaries:
+    name: Build Hubble CLI release binaries
+    needs: check_changes
+    if: ${{ needs.check_changes.outputs.tested == 'true' }}
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Checkout the repository
+        uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
+      - name: Setup go
+        uses: actions/setup-go@0c52d547c9bc32b1aa3301fd7a9cb496313a4491 # v5.0.0
+        with:
+          # renovate: datasource=golang-version depName=go
+          go-version: 1.22.2
+      - name: Build hubble CLI release binaries
+        run: |
+          make -C hubble local-release
+
+  integration-test:
+    name: Run hubble CLI integration tests
+    needs: check_changes
+    if: ${{ needs.check_changes.outputs.tested == 'true' }}
+    runs-on: ubuntu-22.04
+    timeout-minutes: 20
+    steps:
+      - name: Checkout the repository
+        uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
+      - name: Setup go
+        uses: actions/setup-go@0c52d547c9bc32b1aa3301fd7a9cb496313a4491 # v5.0.0
+        with:
+          # renovate: datasource=golang-version depName=go
+          go-version: 1.22.2
+      - name: Build Hubble CLI
+        run: make -C hubble
+      - name: Set up Helm
+        uses: azure/setup-helm@5119fcb9089d432beecbf79bb2c7915207344b78 # renovate: tag=v3.5
+        with:
+          version: ${{ env.HELM_VERSION }}
+      - name: Create kind cluster
+        uses: helm/kind-action@99576bfa6ddf9a8e612d83b513da5a75875caced # v1.9.0
+        with:
+          version: ${{ env.KIND_VERSION }}
+          config: ${{ env.KIND_CONFIG }}
+          cluster_name: kind
+      - name: Deploy Cilium
+        run: |
+          # Deploy cilium with KPR
+          MASTER_IP="$(docker inspect kind-control-plane | jq '.[0].NetworkSettings.Networks.kind.IPAddress' -r)"
+          helm repo add cilium https://helm.cilium.io
+          helm repo update
+          helm install cilium cilium/cilium \
+            --wait \
+            --namespace kube-system \
+            --version ${{ env.CILIUM_VERSION }} \
+            --set kubeProxyReplacement=strict \
+            --set k8sServiceHost="${MASTER_IP}" \
+            --set k8sServicePort=6443 \
+            --set operator.replica=1 \
+            --set hubble.relay.enabled=1
+      - name: Wait for hubble-relay to be running
+        run: |
+          kubectl -n kube-system rollout status deployment/hubble-relay
+      - name: Run integration tests
+        timeout-minutes: 5
+        # work around for hubble CLI thinking we're piping something in via
+        # stdin, even though we aren't
+        shell: 'script -q -e -c "bash --noprofile --norc -eo pipefail {0}"'
```

Comment:
nit (unrelated to this PR, just for a follow-up). This is no longer needed, since Hubble CLI no longer auto-detects stdin.

Reply:
deleted! 🥰